uiowa/uiowa
# Article subhead

Resolves #6147 
Resolves #6150 





# How to test


`git checkout article-subhead && git pull && ddev blt ds --site=default && ddev drush @default.local uli`
Add a new article or edit an existing article
Test out the new subhead field
View the page source and make sure there's only one schema script appearing

Repeat testing on various splits that are built off the article such as periodical feature, now.uiowa.edu, etc. (https://github.com/uiowa/uiowa/find/main).

For Now migration,
`ddev blt ds --site=now.uiowa.edu && ddev drush @now.local en now_migrate -y && ddev drush @now.local uli /admin/config/sitenow/sitenow-migrate`
Complete form
`ddev drush @now.local mim now_news_feature --limit=200`
Check some of the migrated News Feature articles, see that the subhead has been filled in
Check out the Tags and either spot-check a few articles against their source, or check that there are a significant number of Tags including audiences and generic keywords, and that they aren't duplicating.

Transparency: I also updated Point of Contact reference field labels from "Point of Contact(s)" to "Point(s) of Contact" and removed an extra 'point of contact' block from the person display in the now.uiowa.edu site split. Code reviews for these parts are probably adequate.
